tests/e2e/utils/findSnapEvent.js

```diff
@@ -0,0 +1,7 @@
+export function findSnaptrEvent( queue, eventName ) {
```

Comment:
[nitpick] The exported function name doesn't match the file name.

It would also be nice to add some JSDoc info to this function, if possible.